### PR TITLE
LPS-108685 implementing GDPR exporter tests

### PR DESCRIPTION
**Pull request by gabrielibson**

### CODE REVIEW

**Comment by gabrielibson** [comment]
ci:test:sf

**Comment by gabrielibson** [comment]
Related issues:

https://issues.liferay.com/browse/LPS-97802
https://issues.liferay.com/browse/LPS-97804

**Comment by gabrielibson** [comment]
ci:test:forms

**Comment by natocesarrego** [comment]
ci:test:forms

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/EditFormInstanceRecordMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		long ddmFormInstanceRecordId = ParamUtil.getLong(
+			actionRequest, "formInstanceRecordId");
+
+		ddmFormInstanceRecordLocalService.updateFormInstanceRecord(
+			userId, ddmFormInstanceRecordId, false, ddmFormValues,
+			serviceContext);
+	}
+
+	@Reference
+	protected DDMFormInstanceRecordLocalService
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/EditFormInstanceRecordMVCActionCommand.java]: @marcosapmf and @gabrielibson, please change modifier to `private`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/test/DDMFormInstanceRecordUADTestUtil.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+			ServiceContextTestUtil.getServiceContext();
+
+		DDMForm ddmForm = new DDMForm();
+
+		ddmForm.setDefaultLocale(LocaleUtil.US);
+		ddmForm.addAvailableLocale(LocaleUtil.US);
+
+		DDMFormField ddmFormField = new DDMFormField("fieldName", "text");
+
+		ddmForm.addDDMFormField(ddmFormField);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/test/DDMFormInstanceRecordUADTestUtil.java]: @marcosapmf and @gabrielibson, please try to use the existent util methods to create `DDMForm`, `DDMFormField`, etc.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/test/DDMFormInstanceUADTestUtil.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+			ServiceContextTestUtil.getServiceContext();
+
+		DDMForm ddmForm = new DDMForm();
+
+		ddmForm.setDefaultLocale(LocaleUtil.US);
+		ddmForm.addAvailableLocale(LocaleUtil.US);
+
+		DDMFormField ddmFormField = new DDMFormField("fieldName", "text");
+
+		ddmForm.addDDMFormField(ddmFormField);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/test/DDMFormInstanceUADTestUtil.java]: @marcosapmf and @gabrielibson, please try to use the existent util methods to create `DDMForm`, `DDMFormField`, etc.

**Comment by natocesarrego** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/86101. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**